docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -0,0 +1,585 @@
+:linkTroubleshooting: https://github.com/integr8ly/example-customisations/blob/master/docs/troubleshooting.adoc
+:linkGitHubFork: https://help.github.com/articles/fork-a-repo/
+:linkIndexURL: https://github.com/integr8ly/example-customisations/index.adoc
+:walkthrough-1A: *Integrating message-oriented middleware with a RESTful API using AMQ Online*
+:walkthrough-2: *Managing and protecting APIs using API management*
+:publishing-walkthrough: *Adding walkthroughs to a cluster*
+
+= Writing {PRODUCT} Walkthroughs
+
+
+== About Walkthroughs
+
+
+Walkthroughs are tutorials available in {PRODUCT}. The default walkthroughs guide you through the following scenarios:
+
+* {walkthrough-1A}
+* {walkthrough-2}
+
+These walkthroughs are available in the {web-app}.
+
+In addition to the default walkthroughs, you can import existing walkthroughs from a Git repository. 
+The process for adding walkthroughs to your cluster is itself a walkthrough that is available in the {web-app}.
+
+The following sections describe how to write a walkthrough.
+
+To write walkthroughs you require a knowledge of:
```

Comment:
Is there a standard way to call our prerequisites? If I am not mistaken, from the modular doc training, you can use the "Prerequisites" heading.

Reply:
right, I see your point, but it's unreasonable to expect anyone to actually fulfill these as prereqs... that's why I'm going gently here.. maybe 'By the time you have written your first walkthrough, you'll have a knowledge of... ' wdyt?